### PR TITLE
Exception for X25519MLKEM768 naming

### PR DESCRIPTION
**Pull request by kriskwiatkowski**

### CODE REVIEW

**Comment by dconnolly** [comment]
This is not in compliance with [-hybrid-design](https://www.ietf.org/archive/id/draft-ietf-tls-hybrid-design-11.html):

image

File: draft-kwiatkowski-tls-ecdhe-mlkem.md [hunk 1]
```diff
@@ -109,6 +109,10 @@ of the scheme, which will remain more ubiqutous for secp256r1 in the coming year
 For this reason we put the ML-KEM shared secret first in X25519MLKEM768,
 and the ECDH shared secret first in SecP256r1MLKEM768 and SecP384r1MLKEM1024.
 
+Note: The group name X25519MLKEM768 does not adhere to the naming convention outlined in
+{{Section 3.2 of hybrid}}. Specifically, the order of shares in the concatenation has been
+reversed. This is due to historical reasons.
```
Comment (csosto-pk) [inline draft-kwiatkowski-tls-ecdhe-mlkem.md:114]: LGTM

**Review by csosto-pk — APPROVED** [review]

**Review by tomato42 — APPROVED** [review]

**Review by ctz — APPROVED** [review]
This situation is unfortunate but I think this direction leaves us with the minimum amount of mess.

**Comment by kriskwiatkowski** [comment]
Let's merge it as this is current state of the art.
I don't think there is clear consensus regarding changing the name of X25519-MLKEM768. We will get a consensus after document is adopted.

**Pull request merged**